server/util/perms/perms.go

```diff
@@ -46,7 +46,15 @@ func AnonymousUserPermissions() *UserGroupPerm {
 	}
 }
 
-func GroupAuthPermissions(groupID string) *UserGroupPerm {
+func GroupAuthPermissions(u interfaces.UserInfo) *UserGroupPerm {
+	return &UserGroupPerm{
+		UserID:  u.GetUserID(),
+		GroupID: u.GetGroupID(),
+		Perms:   GROUP_READ | GROUP_WRITE,
+	}
+}
```

Comment:
I think we can drop `Auth` from this name since it's redundant with `Permissions`.

maybe @tylerwilliams or @vadimberezniker can weigh on this as well but I wonder if it would make sense to call this `DefaultPermissions` since these perms seem like a pretty sensible default for most things - the object is attributed to the user but we only set Perms bits for the group - which allows the user to later be kicked out of the group and then lose access to the object, while still retaining info about which user created the object.

Reply:
+1 to dropping Auth from the name.

I'm also cool with DefaultPermissions if @vadimberezniker is down

Reply:
Yeah, sounds good to me.